torchgeo/datasets/landcoverai.py

```diff
@@ -50,7 +52,223 @@ class LandCoverAI(NonGeoDataset):
+        mask = sample["mask"].numpy().astype("uint8").squeeze()
+
+        num_panels = 2
+        showing_predictions = "prediction" in sample
+        if showing_predictions:
+            predictions = sample["prediction"].numpy()
+            num_panels += 1
+
+        fig, axs = plt.subplots(1, num_panels, figsize=(num_panels * 4, 5))
+        axs[0].imshow(image)
+        axs[0].axis("off")
+        axs[1].imshow(mask, vmin=0, vmax=4, cmap=self._lc_cmap, interpolation="none")
+        axs[1].axis("off")
+        if show_titles:
+            axs[0].set_title("Image")
+            axs[1].set_title("Mask")
+
+        if showing_predictions:
+            axs[2].imshow(
+                predictions, vmin=0, vmax=4, cmap=self._lc_cmap, interpolation="none"
+            )
+            axs[2].axis("off")
+            if show_titles:
+                axs[2].set_title("Predictions")
+
+        if suptitle is not None:
+            plt.suptitle(suptitle)
+        return fig
+
+
+class LandCoverAIGeo(LandCoverAI, RasterDataset):
+    r"""LandCover.ai Geo dataset.
+
+    See the abstract base LandCoverAI class to find out more.
+    """
+    filename_glob = "images/*.tif"
+    filename_regex = ".*tif"
+    crs = CRS.from_epsg(2180)
+
+    def __init__(
+        self,
+        root: str = "data",
+        res: Optional[float] = None,
+        transforms: Optional[Callable[[Dict[str, Any]], Dict[str, Any]]] = None,
+        cache: bool = True,
+        download: bool = False,
+        checksum: bool = False,
+    ) -> None:
+        """Initialize a new LandCover.ai NonGeo dataset instance.
+
+        Args:
+           root: root directory where dataset can be found
+           res: resolution of the dataset in units of CRS
+                (defaults to the resolution of the first file found)
+           transforms: a function/transform that takes input sample and its target as
+               entry and returns a transformed version
+           cache: if True, cache file handle to speed up repeated sampling
+           download: if True, download dataset and store it in the root directory
+           checksum: if True, check the MD5 of the downloaded files (may be slow)
+
+        Raises:
+           RuntimeError: if ``download=False`` and data is not found, or checksums
+               don't match
+        """
+        LandCoverAI.__init__(self, root, download, checksum)
+        RasterDataset.__init__(
+            self, root, self.crs, res, transforms=transforms, cache=cache
+        )
+
+    def _verify_data(self) -> bool:
+        """Verify if the images and masks are present."""
+        images = os.path.join(self.root, "images", "*.tif")
+        masks = os.path.join(self.root, "masks", "*.tif")
+        return len(glob.glob(images)) > 0 and len(glob.glob(images)) == len(
+            glob.glob(masks)
+        )
+
+    def __getitem__(self, query: BoundingBox) -> Dict[str, Any]:
```

Comment:
This looks like exactly what we want, I'll point contributors to this dataset if they encounter https://github.com/microsoft/torchgeo/discussions/1104#discussioncomment-5001426